letsencrypt/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN apk add --no-cache --update \
         certbot-dns-rfc2136==${CERTBOT_VERSION} \
         certbot-dns-route53==${CERTBOT_VERSION} \
         certbot-dns-sakuracloud==${CERTBOT_VERSION} \
+        certbot-dns-netcup \
```

Comment:
Please pin the version like the other plugins does

Reply:
I did it in the first place but had to remove it again since the add-on won't be built with the pinned version:

`20-02-04 07:21:56 ERROR (SyncWorker_1) [hassio.docker.addon] Can't build local/amd64-addon-letsencrypt:4.3: The command '/bin/ash -o pipefail -c apk add --no-cache --update         libffi         musl         openssl     && apk add --no-cache --virtual .build-dependencies         g++         libffi-dev         musl-dev         openssl-dev     && pip3 install --no-cache-dir certbot==${CERTBOT_VERSION}         certbot-dns-cloudflare==${CERTBOT_VERSION}         certbot-dns-cloudxns==${CERTBOT_VERSION}         certbot-dns-digitalocean==${CERTBOT_VERSION}         certbot-dns-dnsimple==${CERTBOT_VERSION}         certbot-dns-dnsmadeeasy==${CERTBOT_VERSION}         certbot-dns-gehirn==${CERTBOT_VERSION}         certbot-dns-google==${CERTBOT_VERSION}         certbot-dns-linode==${CERTBOT_VERSION}         certbot-dns-luadns==${CERTBOT_VERSION}         certbot-dns-nsone==${CERTBOT_VERSION}         certbot-dns-ovh==${CERTBOT_VERSION}         certbot-dns-rfc2136==${CERTBOT_VERSION}         certbot-dns-route53==${CERTBOT_VERSION}         certbot-dns-sakuracloud==${CERTBOT_VERSION}         certbot-dns-netcup==${CERTBOT_VERSION}     && apk del .build-dependencies' returned a non-zero code: 1
20-02-04 07:21:56 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished coro=<AddonManager.rebuild() done, defined at /usr/src/hassio/hassio/addons/__init__.py:219> exception=AddonsError()>`

Reply:
Look to pypi and pin the version they you want like the corebot version is pined